sdk/servicebus/service-bus/test/sendBatch.spec.ts

```diff
@@ -0,0 +1,663 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import chai from "chai";
+const should = chai.should();
+import chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
+import { ServiceBusMessage, ReceivedMessage } from "../src";
+import { TestClientType } from "./utils/testUtils";
+import { Receiver } from "../src/receivers/receiver";
+import { ServiceBusClientForTests, createServiceBusClientForTests } from "./utils/testutils2";
+import { Sender } from "../src/sender";
+
+describe("Send Batch", () => {
+  let senderClient: Sender;
+  let receiverClient: Receiver<ReceivedMessage>;
+  let serviceBusClient: ServiceBusClientForTests;
+  interface EntityName {
+    queue?: string | undefined;
+    topic?: string | undefined;
+    subscription?: string | undefined;
+  }
+  let entityNames: EntityName;
+
+  before(() => {
+    serviceBusClient = createServiceBusClientForTests();
+  });
+
+  after(() => {
+    return serviceBusClient.test.after();
+  });
+
+  async function beforeEachTest(entityType: TestClientType): Promise<void> {
+    entityNames = await serviceBusClient.test.createTestEntities(entityType);
+
+    senderClient = serviceBusClient.test.addToCleanup(
+      serviceBusClient.getSender(entityNames.queue ?? entityNames.topic!)
+    );
+  }
+
+  async function afterEachTest(): Promise<void> {
+    await senderClient.close();
+  }
+
+  async function receiveAllMessages(
```

Comment:
Let's add this to the `servicebusClient.test` class.

Also, I think you can re-use or just call some of the methods in there to do the same thing.

Reply:
This is not optimized well enough for receiving from a single session.
The previous existing cleanup scenarios can't handle receiving from multiple sessions, so I had add this by opening and closing receivers for multiple sessions sequentially.

Reply:
I'll see, compare, split the methods and keep what is required and make sure there is no duplication.

Reply:
Added to `servicebusClient.test` with some appropriate updates.